plugins/modules/win_xml.ps1

```diff
@@ -265,7 +265,7 @@ elseif ($type -eq "attribute") {
             }
             else {
                 # element node processing
-                if ($node.Name -eq $attribute ) {
+                if ($node.HasAttribute($attribute)) {
```

Comment:
Please create a [changelog fragment](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment). See [this fragment](https://github.com/ansible/ansible/blob/c2f528b7686c9573aa8493357219e0bbe4416dd3/changelogs/fragments/reboot-change-default-boot-command.yaml) as an example. Could you please add a test for this change? Thanks in advance